mmdet/datasets/pipelines/transforms.py

```diff
@@ -494,6 +496,279 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class MosaicCrop(object):
```

Comment:
The implementation seems to be very similar to `RandomCrop` in this file. Can we directly use the existing one?

Reply:
They are somewhat different. For example, MosaicCrop can crop out of the image and fill in the blank pixels with mean/std.

Reply:
Then this could be a combination of RandomExpand and RandomCrop.